src/azul/chalice.py

```diff
@@ -141,6 +141,8 @@ def __init__(self,
         self.unit_test = unit_test
         self.non_interactive_routes: set[tuple[str, str]] = set()
         assert 'paths' not in spec, 'The top-level spec must not define paths'
+        # Inject contact details into Swagger spec
+        spec['info']['description'] += config.contact_us
```

Comment:
Let's write this so that it creates these keys if they're absent. I think our `deep_dict_merge` may be helpful.

Also, the `spec` parameter is declared `JSON` so it shouldn't be modified in place.